workflow/profile/slurm/config.yaml

```diff
@@ -39,6 +39,11 @@ set-resources:
     mem: 179GB
     gpus_per_node: ""
     cpus_per_task: 1
+  correlations:
+    partition: gpu
+    mem: 179GB
+    gpus_per_node: tesla_v100s:1
```

Comment:
Is this necessary? We should avoid forcing runs on V100's unless needed (memory/time).

Reply:
The correlations rule performs forward-passes with the filtered data and should run on GPU.
The correlation calculations on the model outputs doesn't use the GPU though, so there may be a more optimal solution.